### PR TITLE
Add kubelet OVAL checks

### PR DESCRIPTION
**Pull request by redhatrises**

### CODE REVIEW

**Comment by redhatrises** [comment]
Build is expected to fail. Need #3543 merged.

**Comment by shawndwells** [comment]
@openscap-ci retest this please

File: applications/openshift/kubelet/kubelet_enable_client_server_rotation/rule.yml [hunk 1]
```diff
@@ -6,7 +6,9 @@ description: |-
     To enable the kubelet to rotate server certificates, edit the kubelet configuration
     file <tt>/etc/origin/node/node-config.yaml</tt>
     on the kubelet node(s) and set the below parameter:
-    <pre>RotateKubeletServerCertificate=true</pre>
+    <pre>kubeletArguments:
+      feature-gates:
+        - RotateKubeletClientCertificate=true,RotateKubeletServerCertificate=true</pre>
```
Comment (shawndwells) [inline applications/openshift/kubelet/kubelet_enable_client_server_rotation/rule.yml]: should this be only for client?

File: applications/openshift/kubelet/kubelet_enable_client_server_rotation/rule.yml [hunk 2]
```diff
@@ -18,7 +20,7 @@ ocil_clause: 'the kubelet cannot rotate server certificate'
 
 ocil: |-
     Run the following command on the kubelet node(s):
-    <pre>$ sudo grep -B1 RotateKubeletServerCertificate /etc/origin/node/node-config.yaml</pre>
+    <pre>$ sudo grep RotateKubeletServerCertificate /etc/origin/node/node-config.yaml</pre>
```
Comment (shawndwells) [inline applications/openshift/kubelet/kubelet_enable_client_server_rotation/rule.yml]: For consistency with ``description`` and the rule name, should this be client or server?

Comment (redhatrises) [reply applications/openshift/kubelet/kubelet_enable_client_server_rotation/rule.yml]: Mmmmm.... What would be the best way of handling multiple configs in one line? If you look at what we have traditionally done with the `pam` settings like `pam_faillock`, we show all options.
Having `....,OPTION,....` might be more confusing. Open to ideas.

File: applications/openshift/kubelet/kubelet_enable_client_server_rotation/rule.yml [hunk 3]
```diff
@@ -6,7 +6,9 @@ description: |-
     To enable the kubelet to rotate server certificates, edit the kubelet configuration
     file <tt>/etc/origin/node/node-config.yaml</tt>
     on the kubelet node(s) and set the below parameter:
```
Comment (shawndwells) [inline applications/openshift/kubelet/kubelet_enable_client_server_rotation/rule.yml]:
```suggestion
    on the kubelet node(s) and ensure <tt>RotateKubeletServerCertificate</tt> is included
    in the <tt>feature-gates</tt> arguments:
```

Comment (shawndwells) [reply applications/openshift/kubelet/kubelet_enable_client_server_rotation/rule.yml]: @redhatrises how about this?

Comment (redhatrises) [reply applications/openshift/kubelet/kubelet_enable_client_server_rotation/rule.yml]: That would work. Although we lose showing how to set it in the description.

Comment (shawndwells) [reply applications/openshift/kubelet/kubelet_enable_client_server_rotation/rule.yml]: Not following how that'd be lost =/

Comment (redhatrises) [reply applications/openshift/kubelet/kubelet_enable_client_server_rotation/rule.yml]: I reread. I was mistaken on it.

Comment (redhatrises) [reply applications/openshift/kubelet/kubelet_enable_client_server_rotation/rule.yml]: @shawndwells updated.

File: applications/openshift/kubelet/kubelet_enable_client_server_rotation/rule.yml (hunk 1 above)
Comment (redhatrises) [reply applications/openshift/kubelet/kubelet_enable_client_server_rotation/rule.yml]: Came up with a different description that resolved this questions.

File: applications/openshift/kubelet/kubelet_enable_client_server_rotation/rule.yml (hunk 2 above)
Comment (redhatrises) [reply applications/openshift/kubelet/kubelet_enable_client_server_rotation/rule.yml]: rule name updated to match the client rule name.

**Comment by shawndwells** [comment]
Now lets wait a million years for jenkins

**Comment by shawndwells** [comment]
(seriously, wonder if we should replatform jenkins to something with more resources?)

**Pull request merged**